static/js/components/EmploymentTab.js

```diff
@@ -1,21 +1,131 @@
 import React from 'react';
+import Button from 'react-mdl/lib/Button';
+import Grid from 'react-mdl/lib/Grid';
+import { Cell } from 'react-mdl/lib/Grid';
+
+import ProfileTab from "../util/ProfileTab";
+import { saveAndContinue } from '../util/profile_edit';
+
+class EmploymentTab extends ProfileTab {
+  constructor(props) {
+    super(props);
+    this.saveAndContinue = saveAndContinue.bind(this, '/dashboard', this.validation);
+    this.blankWorkHistoryEntry = {
+      id: null,
+      position: "",
+      industry: "",
+      company_name: "",
+      start_date: "",
+      end_date: "",
+      city: "",
+      country: "",
+    }
+  }
+
+  validation (profile, requiredFields) {
+    const keys = [
+      'position',
+      'industry',
+      'company_name',
+      'start_date',
+      'end_date',
+      'city',
+      'country'
+    ];
+
+    let nestedFields = (index) => {
+      let keySet = (key) => ['work_history', index, key];
+      return keys.map(key => keySet(key));
+    }
+
+    return requiredFields.concat(
+      ...profile.work_history.map( (v,i) => nestedFields(i))
+    );
+  }
+
+
+  employmentEntriesForm (index) {
+    let keySet = (key) => ['work_history', index, key];
+    return (
+      <Grid key={index} className="profile-tab-grid">
+        <Cell col={12}>
+          {this.boundTextField(keySet('company_name'), 'Company Name')}
+        </Cell>
+        <Cell col={12}>
+          {this.boundTextField(keySet('position'), 'Position')}
+        </Cell>
+        <Cell col={12}>
+          {this.boundTextField(keySet('industry'), 'Industry')}
+        </Cell>
+        <Cell col={12}>
+          {this.boundTextField(keySet('city'), 'City')}
+        </Cell>
+        <Cell col={12}>
+          {this.boundSelectField(keySet('country'), 'Country', this.countryOptions)}
+        </Cell>
+        <Cell col={12}>
+          {this.boundDateField(keySet('start_date'), 'Start Date')}
+        </Cell>
+        <Cell col={12}>
+          {this.boundDateField(keySet('end_date'), 'End Date')}
+        </Cell>
+      </Grid>
+    );
+  }
 
-class EmploymentTab extends React.Component {
   render () {
+    this.employmentEntriesForm = this.employmentEntriesForm.bind(this);
```

Comment:
Can you create a local variable here instead of reassigning since this variable is only used inside the render method?